packages/fluentui/react-bindings/src/hooks/useStyles.ts

```diff
@@ -9,12 +9,12 @@ import {
 import * as React from 'react';
 // @ts-ignore We have this export in package, but it is not present in typings
 import { ThemeContext } from 'react-fela';
+import { noopRenderer } from '@fluentui/styles';
```

Comment:
I feel like we need a `@fluentui/react-northstar-renderer` package, or something similar...

Reply:
Same feeling, but it looks strange to introduce a package for a single file...

Reply:
I think it would be nice if we can extract to a separate package: typings for the renderer + default implementation (even if it is the noopRenderer).. But anyway, not a deal breaker..